extractor/text_test.go

```diff
@@ -67,46 +68,47 @@ func TestTextExtractionFragments(t *testing.T) {
         BT
         /UniDocCourier 24 Tf
         (Hello World!)Tj
-        0 -10 Td
-        (Doink)Tj
-        ET
-        `,
-			text: "Hello World!\nDoink",
-		},
-		{
-			name: "landscape",
-			contents: `
-        BT
-        /UniDocCourier 24 Tf
-        0 1 -1 0 0 0 Tm
-        (Hello World!)Tj
-        0 -10 Td
-        (Doink)Tj
-        ET
-        `,
-			text: "Hello World!\nDoink",
-		},
-		{
-			name: "180 degree rotation",
-			contents: `
-        BT
-        /UniDocCourier 24 Tf
-        -1 0 0 -1 0 0 Tm
-        (Hello World!)Tj
-        0 -10 Td
+        0 -25 Td
         (Doink)Tj
         ET
         `,
 			text: "Hello World!\nDoink",
 		},
+		// TODO(peterwilliams97): Reinstate rotated text tests.
```

Comment:
Probably want to put these back in

Reply:
I reinstated all the old functionality. All tests are back in text_test.go.